### PR TITLE
Fix header background width

### LINKED ISSUE #26: Header background doesn't fill width at certain resolutions
image

### PR DESCRIPTION
**Pull request by R-Jim**

### CODE REVIEW

**Comment by R-Jim** [comment]
Result
image
image

**Review by DuongTyler — APPROVED** [review]
can't remember if there was a reason it was laid out like this but otherwise looks good to me

File: frontend/src/components/headerSection/header.css [hunk 1]
```diff
@@ -85,7 +82,6 @@
 @media (max-width: 420px /*naisu*/) {
     .app-header {
         min-height: 50vh;
-        background-size: 100%;
```
Comment (zapplecat) [inline frontend/src/components/headerSection/header.css]: This was intentional to make sure mobile views don't see a half cut off background.

https://github.com/Manotomo-Alliance-Support-Squad/WWS/issues/26#issuecomment-848002005

**Comment by zapplecat** [comment]
Also, looks like the commit isn't signed somehow...?

File: frontend/src/components/headerSection/header.css (hunk 1 above)
Comment (R-Jim) [reply frontend/src/components/headerSection/header.css]: Updated

**Pull request merged**